ocw/models.py

```diff
@@ -73,10 +72,19 @@ class CspInfo(models.Model):
     type = models.CharField(max_length=200)
 
     def get_openqa_job_link(self):
-        if self.get_tag('openqa_created_by') == 'openqa-suse-de' and self.get_tag('openqa_var_JOB_ID') is not None:
-            url = f"{PCWConfig.get_feature_property('webui', 'openqa_url')}/t{self.get_tag('openqa_var_JOB_ID')}"
+        server = self.get_tag('openqa_var_SERVER')
+        job_id = self.get_tag('openqa_var_JOB_ID')
+        if server and job_id:
+            url = f"{server}/t{job_id}"
             title = self.get_tag('openqa_var_NAME', '')
             return {'url': url, 'title': title}
+        created_by = self.get_tag('created_by')
+        if created_by is not None:
+            if "/t" in created_by:
```

Comment:
why ? which flow we supporting here ?

Reply:
> why ? which flow we supporting here ?

The `$provider_cli`  tests.

Reply:
but `created_by` should not contain test url in any case so why we have test which expects something like this ? maybe better tweak/drop such test instead of aligning prod code with it ?

Reply:
> but `created_by` should not contain test url in any case so why we have test which expects something like this ? maybe better tweak/drop such test instead of aligning prod code with it ?

Then I would do that in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/17420

It does contain URL's:

```
$ grep -r created_by tests/publiccloud/
tests/publiccloud/aws_cli.pm:    my $created_by = "$openqa_url/t$job_id";
tests/publiccloud/aws_cli.pm:    my $tag = "{Key=openqa-cli-test-tag,Value=$job_id},{Key=openqa_created_by,Value=$created_by},{Key=openqa_ttl,Value=$openqa_ttl}";
tests/publiccloud/azure_cli.pm:    my $created_by = "$openqa_url/t$job_id";
tests/publiccloud/azure_cli.pm:    my $tags = "openqa-cli-test-tag=$job_id openqa_created_by=$created_by openqa_ttl=$openqa_ttl";
tests/publiccloud/azure_more_cli.pm:    my $created_by = "$openqa_url/t$job_id";
tests/publiccloud/azure_more_cli.pm:    my $tags = "openqa-cli-test-tag=$job_id openqa_created_by=$created_by openqa_ttl=$openqa_ttl";
tests/publiccloud/google_cli.pm:    my $labels = "openqa-cli-test-label=$job_id,openqa_created_by=$openqa_hostname,openqa_ttl=$openqa_ttl"
```